### PR TITLE
Add recipe for sshtunnel

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,68 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (jakirkham) [inline recipes/example/meta.yaml]: Please restore the example.

Comment (basnijholt) [inline recipes/example/meta.yaml]: Restored it.

**Comment by jakirkham** [comment]
So the example appears to have turned into your recipe. 😕 Please just checkout the `example` from `master` (after updating your copy of `master`).

File: recipes/sshtunnel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+source:
+  fn: {{ version }}.zip
+  url: https://github.com/pahaz/sshtunnel/archive/{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: 
+   - python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipes/sshtunnel/meta.yaml]: Please move to same line as `script` and drop `-`.

Comment (jakirkham) [inline recipes/sshtunnel/meta.yaml]: Also please add the following.

```yaml
  entry_points:
    - sshtunnel = sshtunnel:_cli_main
```

Comment (basnijholt) [inline recipes/sshtunnel/meta.yaml]: Done :)

**Comment by jakirkham** [comment]
Thanks @basnijholt.

**Pull request merged**